### PR TITLE
SLES-15-030720 - fixes

### PR DESCRIPTION
**Pull request by rumch-se**
#### Description:

- _SLES-15-030720 review required_

#### Rationale:

- _Implementation of changes proposed by upstream_

- Fixes # _Issue number here (e.g. #26) or remove this line if no issue exists._

### CODE REVIEW

File: shared/templates/audit_rules_syscall_events/template.yml
Comment (Mab879) [inline shared/templates/audit_rules_syscall_events/template.yml:2]: Please remove these lines. This appears to be part of why the build is failing on this branch.

**Review by Mab879 — CHANGES_REQUESTED** [review]

File: shared/templates/audit_rules_syscall_events/template.yml
Comment (rumch-se) [reply shared/templates/audit_rules_syscall_events/template.yml]: Hello @Mab879 - I have removed them.

**Comment by rumch-se** [comment]
Hello @Mab879 

I have two PRs - this one and https://github.com/ComplianceAsCode/content/pull/7455 which have are not merged yet. I opened them in August 2021. These rules have logic similar to the rules (already merged) included into https://github.com/ComplianceAsCode/content/pull/7529.

Is there any particular reason for this delay and how can I help to unblock the merge?

Have a great day
Rumen

**Comment by Mab879** [comment]
Sorry for the delay. I'm reviewing this one as we speak and as for #7455 I have requested a review from @jan-cerny as he had some comments.

**Comment by Mab879** [comment]
Tests pass in a VM.

```
$ ./test_suite.py rule --libvirt qemu:///system test_ssg_rhel84_auto --datastream ../build/ssg-rhel8-ds.xml audit_rules_unsuccessful_file_modification_renameat
Setting console output to log level INFO
INFO - The base image option has not been specified, choosing libvirt-based test environment.
INFO - Logging into /home/mburket/code/ComplianceAsCode/content/tests/logs/rule-custom-2021-11-11-1346/test_suite.log
INFO - xccdf_org.ssgproject.content_rule_audit_rules_unsuccessful_file_modification_renameat
INFO - Script correct_rules.pass.sh using profile (all) OK
INFO - Script only_eacces.fail.sh using profile (all) OK
INFO - Script rules_not_there.fail.sh using profile (all) OK
INFO - Script wrong_list_action.fail.sh using profile (all) OK
INFO - Script wrong_syscall.fail.sh using profile (all) OK
INFO - Script one_filter.fail.sh using profile (all) OK

```

**Review by Mab879 — APPROVED** [review]

**Pull request merged**
